python/ray/scripts/scripts.py

```diff
@@ -466,7 +466,6 @@ def debug(address):
 )
 @click.option(
     "--temp-dir",
-    hidden=True,
     default=None,
     help="manually specify the root temporary dir of the Ray process",
```

Comment:
Can you also mention this should be used with `--head` flag?

Reply:
Done